### ISSUE TITLE
Webpack aliases not working 

### ISSUE THREAD

**Issue by theharshin** [nklayman/vue-cli-plugin-electron-builder#55]
**Describe the bug**
The webpack aliases created in vue.config.js are not being resolved.

**To Reproduce**
- Create a project with Vue CLI UI
- Install vue-cli-plugin-electron-builder via terminal
- Create a vue.config.js and add aliases
- Run the command `npm run serve:electron`

**Expected behavior**
The same alias configuration works with `npm run serve` & build is being created successfully. It fails with this plugin only.

**Screenshots**
*npm run serve:electron*
![image](https://user-images.githubusercontent.com/7118937/44447491-fb8a6b00-a605-11e8-89a6-b8c412c11e6e.png)

------
*npm run serve*
![image](https://user-images.githubusercontent.com/7118937/44447531-1eb51a80-a606-11e8-9604-175a5ec5305d.png)

**Environment (please complete the following information):**
 - OS and version: macOS Mojave
 - Node version: 10.6.0
 - npm version: 6.2.0
 - vue-cli-plugin-electron-builder version : 1.0.0-beta.8

**Comment by theharshin** [comment]
**vue.config.js**

```
module.exports = {
  configureWebpack: {
    resolve: {
      alias: {
        Mixins: path.resolve(__dirname, "./src/mixins"),
        Components: path.resolve(__dirname, "./src/components"),
        Images: path.resolve(__dirname, "./src/assets/img/"),
        Styles: path.resolve(__dirname, "./src/assets/scss/"),
        Interfaces: path.resolve(__dirname, "./src/interfaces"),
        Views: path.resolve(__dirname, "./src/views"),
        Bus: path.resolve(__dirname, "./src/bus.js")
      }
    }
  },
};
```

**Comment by theharshin** [comment]
I've drilled down the issue and this is what I found:

The plugin is using cli-service 3.0.0-rc.2 & latest has 3.0.1
In `rc` version, the `serve` function will use additional configuration if provided and in latest, that is not the case.
![image](https://user-images.githubusercontent.com/7118937/44457170-2386c800-a620-11e8-8bc3-13e25b66541d.png)

Now in plugin, the`config` has been passed as additional parameter and thus the default webpack config is being overwritten and aliases are lost.
![image](https://user-images.githubusercontent.com/7118937/44457533-161e0d80-a621-11e8-97c9-c5fd7aef001d.png)

**Comment by theharshin** [comment]
This is what I've used as workaround for now:

![image](https://user-images.githubusercontent.com/7118937/44457980-68136300-a622-11e8-9be2-dc884b6fd687.png)

**Comment by nklayman** [comment]
Set vcp-electron-builder's version to nklayman/vcp-electron-builder. This will have it pull from the github repo, which uses the latest Vue CLI.

**Comment by nklayman** [comment]
This [commit](https://github.com/nklayman/vue-cli-plugin-electron-builder/commit/c068159eebe903989c0c8955b0843d568a1918d6) removes the need for the modified vue-cli-service. It will be released soon with 1.0.0-rc.1.

**Comment by nklayman** [comment]
1.0.0-rc.1 has landed which fixes this issue. Simply upgrade to it and your alias will work fine.
